relevanceai/operations/cluster/cluster.py

```diff
@@ -131,7 +129,7 @@ def _get_package(self, model):
         elif "hdbscan" in model_name:
             package = "hdbscan"
 
-        elif "community_detection" in model_name:
+        elif "communitydetection" in model_name:
```

Comment:
huh

Reply:
Model name has underscores recovered at beginning of func